### PR TITLE
Drop all mentions of php 8.0

### PR DESCRIPTION
**Pull request by st3iny**
### ☑️ Resolves

* Follow-up to https://github.com/nextcloud/server/pull/44607

#### Admin manual (significant changes)

grafik

grafik

#### Developer manual

grafik

### CODE REVIEW

File: admin_manual/installation/system_requirements.rst [hunk 1]
```diff
@@ -31,8 +31,7 @@ For best performance, stability and functionality we have documented some recomm
 | Webserver        | - **Apache 2.4 with** ``mod_php`` **or** ``php-fpm`` (recommended)    |
 |                  | - nginx with ``php-fpm``                                              |
 +------------------+-----------------------------------------------------------------------+
-| PHP Runtime      | - 8.0 (*deprecated*)                                                  |
-|                  | - 8.1                                                                 |
+| PHP Runtime      | - 8.1 (*deprecated*)                                                  |
 |                  | - **8.2** (*recommended*)                                             |
 |                  | - 8.3                                                                 |
```
Comment (nickvergessen) [inline admin_manual/installation/system_requirements.rst:36]: We can recommend 8.3 by now
```suggestion
|                  | - 8.2                                                                 |
|                  | - **8.3** (*recommended*)                                             |
```

File: admin_manual/issues/general_troubleshooting.rst [hunk 2]
```diff
@@ -171,7 +171,7 @@ Web server and Nextcloud itself.
    Linux distros or operating systems they can differ.
 
 * The logfile of Apache2 is located in ``/var/log/apache2/error.log``.
-* The logfile of PHP can be configured in your ``/etc/php/8.0/apache2/php.ini``.
+* The logfile of PHP can be configured in your ``/etc/php/8.1/apache2/php.ini``.
```
Comment (nickvergessen) [inline admin_manual/issues/general_troubleshooting.rst:174]: lets use 8.3 to avoid touching it every year? :P

**Comment by tflidd** [comment]
For NC 30, they will have released Ubuntu 24.04 LTS (planned for April 2024). Do we want to add it as well (not sure about implications for tests)?

**Comment by nickvergessen** [comment]
> For NC 30, they will have released Ubuntu 24.04 LTS (planned for April 2024). Do we want to add it as well (not sure about implications for tests)?

We can only test/add it once it's there.
So this will be a follow up.

The PHP 8.0 drop should have happened with 28 actually, but Suse Linux Enterprise was not ready in time

File: admin_manual/installation/system_requirements.rst (hunk 1 above)
Comment (st3iny) [reply admin_manual/installation/system_requirements.rst]: Done

File: admin_manual/issues/general_troubleshooting.rst (hunk 2 above)
Comment (st3iny) [reply admin_manual/issues/general_troubleshooting.rst]: Done

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_30.rst
[the remark this answers is not in the document]
Comment (st3iny) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_30.rst:1]: Done

**Review by nickvergessen — APPROVED** [review]

**Review by susnux — APPROVED** [review]

**Comment by AndyScherzinger** [comment]
PHP 8.0 drop needs to only happen for 30.

Community read this, i.e. @dartcafe and dropped PHP8.0 for 28+

So this is Nc30 only, but latest docs show this change !

**Comment by susnux** [comment]
> So this is Nc30 only, but latest docs show this change !

I think the problem is that docs for 30 and 29 both link to *latest*.
Maybe instead we should make 29 link to *stable* (as the latest stable release) and 30 to *latest* as the latest (un)stable release.

**Comment by AndyScherzinger** [comment]
To which URL are the documentation link pointing within a 29 instance?

**Comment by susnux** [comment]
> To which URL are the documentation link pointing within a 29 instance?

They are always link to the number version, e.g.: "https://docs.nextcloud.com/server/29/admin_manual/"
And this correctly still lists PHP 8.0.

Only https://docs.nextcloud.com/ is linking to the wrong documentation.

**Pull request merged**